core/test/computed/viewport-meta-test.js

```diff
@@ -69,16 +74,20 @@ describe('ViewportMeta computed artifact', () => {
     ];
 
     await Promise.all(viewports.map(async viewport => {
-      const {isMobileOptimized} =
+      const {isMobileOptimized, rawContentString} =
         await ViewportMeta.compute_(makeMetaElements(viewport));
       assert.equal(isMobileOptimized, true);
+      assert.equal(rawContentString, viewport);
     }));
   });
 
   it('recognizes interactive-widget property', async () => {
     const viewport = 'width=device-width, interactive-widget=resizes-content';
-    const {parserWarnings} = await ViewportMeta.compute_(makeMetaElements(viewport));
-    assert.equal(parserWarnings[0], undefined);
```

Comment:
driveby change of `assert.equal(parserWarnings[0], undefined)` to `assert.equal(parserWarnings.length, 0)` because the first one seems like it's asserting that there is at least one warning, but its value is `undefined`. Not sure if this was important at the time, but seems correct now, at least (happened in https://github.com/GoogleChrome/lighthouse/pull/14664 @connorjclark, maybe it had to do with the `metaviewport-parser` release workaround or something?)